terraform/e2e/variables.tf

```diff
@@ -57,15 +57,19 @@ variable "bigquery_table_delete_protection" {
   default     = false
 }
 
-variable "pmap_mapping_default_resource_scope" {
+variable "service_specific_container_env_vars" {
+  type        = map(string)
   description = <<EOT
-        "The scope for where the resources resides in.
+        "The extra envirnoment variables for mapping services as key value pair.
```

Comment:
Update the description. It can be used for both mapping and policy. But you can use mapping service as an example.